Fakebook.Posts/Fakebook.Posts.Domain/Models/Comment.cs

```diff
@@ -17,9 +16,17 @@ public Comment(string userEmail, string content)
         {
             if (string.IsNullOrWhiteSpace(userEmail)) throw new ArgumentException("User email is required.", nameof(userEmail));
             if (string.IsNullOrWhiteSpace(content)) throw new ArgumentException("Post content is required.", nameof(content));
+            try
+            {
+                userEmail = new System.Net.Mail.MailAddress(userEmail).Address;
```

Comment:
why not have `using System.Net.Mail;` at the top instead of having the full identifier here

Reply:
added name space to the top as suggested